webview/src/setup/components/dvc/CliUnavailable.tsx

```diff
@@ -29,6 +33,9 @@ export const CliUnavailable: React.FC<PropsWithChildren> = ({ children }) => {
       </p>
       <div className={styles.sideBySideButtons}>
         <Button onClick={installDvc} text="Install (pip)" />
+        {isPythonExtensionUsed && (
+          <Button onClick={updatePythonEnvironment} text="Update Env" />
```

Comment:
This pr's focus is adding the logic and quick pick for updating the python environment or creating a new one. I'm going to do separate prs for:

* adding a `--user` to the command if the environment is global
* improving the step's contents (for example, updating the text for "Configure")

We can merge this as is or I'll add a `do not merge` label until the other two prs are ready :)